models/staticLinks.js

```diff
@@ -9,7 +9,15 @@ module.exports = {
       status: String,
       appType: String,
       metaInformation: Object,
-      isDeleted: Boolean
+      isDeleted: {
+        type : Boolean,
+        default : false
+      },
+      appName : String,
+      isCommon : {
```

Comment:
`title : { 
      type : String,
      default: function() {
        if (this.submissionNumber && this.submissionNumber > 0) {
          return "Observation "+this.submissionNumber;
        } else {
          return "Observation";
        }
      }
    }`